src/lib/rnp.c

```diff
@@ -167,14 +167,14 @@ resolve_userid(rnp_t *rnp, const rnp_key_store_t *keyring, const char *userid)
         userid += 2;
     }
     io = rnp->io;
-    if ((key = rnp_key_store_get_key_by_name(io, keyring, userid)) == NULL) {
+    if (rnp_key_store_get_key_by_name(io, keyring, userid, &key) != RNP_OK) {
         (void) fprintf(io->errs, "cannot find key '%s'\n", userid);
     }
     return key;
 }
 
 /* return 1 if the file contains ascii-armoured text */
```

Comment:
so, 1 means armoured, 0 means not-armoured and -1 error. would be good to update comment